### ISSUE TITLE
Tree subsampling within workflow

### ISSUE THREAD

**Issue by btoup15** [TreeScaper/CloudforestDocker#1]
Seems like right now, unless a specific method of subsampling is specified in a premade workflow that that option can't be changed when a user goes to do their own run with the workflow. I think having "no subsampling" is probably best for the default in most instances, but that should be changeable in pre-established workflows based on whether or not the user wants subsampling in that instance.

**Comment by tmcgowan** [comment]
I had not noticed this behavior. It looks like anytime a choice is a conditional, the ability to edit at runtime is not enabled. I can see the same issue with NLDR input type (another conditional). I will have to look more deeply into Galaxy itself to see if I can get around the lack of runtime editing.
@reid-wagner @btoup15

**Comment by tmcgowan** [comment]
I spent some time in the galaxy git issues, and this behavior is by design. It makes sense, workflows are for pre-packaged hands off use. Being able to edit a conditional could get the user into trouble if there are conflicting parameters. However, the subsampling does not have to be a conditional. I did it that way to make the UI clean, but it is not necessary. I am working on a change that will allow for user editing in a workflow. I can see the utility of  allowing a user to subsample on a workflow so they can get a sense of the data, then run the workflow with no subsampling.
